catalog/ui/catalog-ui-search/package.json

```diff
@@ -5,9 +5,10 @@
   "version": "0.0.1",
   "license": "LGPL-3.0",
   "scripts": {
-    "start": "grunt start",
+    "start": "grunt start --webpackConfig=./webpack/config/dev.js",
```

Comment:
This doesn't look right to me. Won't this force webpack to always run with the dev config? I thought we wanted prod to be the default behavior.